examples/mobile/UIComponents/webclip/link-tab/webclip.html

```diff
@@ -7,10 +7,102 @@
 			<button data-icon="reorder"></button>
 		</div>
 	</div>
-	<div class="ui-content">
-		<button data-inline="true">Tab</button>
-		<button data-inline="true">Tab</button>
-		<button data-inline="true">Tab</button>
+	<div class="ui-content ui-tabs" style="height: 215px;">
+		<div class="ui-sub-tab ui-sub-tab-static">
+			<ul>
+				<li class="ui-li-anchor">
+					<a class="ui-tab-active" href="#">
+						One
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="#">
+						Two
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="#">
+						Three
+					</a>
+				</li>
+			</ul>
+		</div>
+		<div class="ui-section-changer">
+			<div>
+				<section class="ui-content ui-section-active">
+					<ul class="ui-listview ui-content-area">
+						<li class="ui-li-divider">
+							<div class="ui-li-text">
+								<span class="ui-li-text-title">
+									Main text 01
+								</span>
+								<span class="ui-li-text-sub ui-li-text-value">
+									Value text
+								</span>
+							</div>
+						</li>
+						<li class="ui-li-divider">
+							<div class="ui-li-text">
+								<span class="ui-li-text-title">
+									Main text 02
+								</span>
+							</div>
+						</li>
+					</ul>
+				</section>
+				<section class="ui-content">
+					<ul class="ui-listview ui-content-area">
+						<li class="ui-li-has-icon ui-li-divider">
+							<div class="ui-li-icon">
+								<img src="images/tw_list_icon_connections.svg"/>
+							</div>
+							<div class="ui-li-text">
+								<span class="ui-li-text-title">
+									Main text 01
+								</span>
+								<span class="ui-li-text-sub">
+									This screen is not settings app.
+								</span>
+							</div>
+						</li>
+						<li class="ui-li-has-icon">
+							<div class="ui-li-icon">
+								<img src="images/thumbnail_013.jpg"/>
+							</div>
+							<div class="ui-li-text">
+								<span class="ui-li-text-title">
+									Main text 02
+								</span>
+								<span class="ui-li-text-sub">
+									It's focused case of icon+text list
+								</span>
+							</div>
+						</li>
+					</ul>
+				</section>
+				<section class="ui-content">
+					<ul class="ui-listview ui-content-area">
+						<li class="ui-li-divider">
+							<div class="ui-li-text">
+								<span class="ui-li-text-title">
+									Main text 03
+								</span>
+								<span class="ui-li-text-sub">
+									This is the second line for descriptive phrases. Chunking background color is #fcfcfc.
```

Comment:
better don't talk about colors in sample texts. Color can be changed by theme change

Reply:
Done.